commerce_coordinator/apps/titan/serializers.py

```diff
@@ -0,0 +1,26 @@
+""" Serializers for data manipulated by commerce-coordinator titan API endpoints. """
+
+from rest_framework import serializers
+
+
+class OrderFulfillSerializer(serializers.Serializer):
+    """ Serializer to use with instances of OrderFulfillView for order fulfillment from Titan """
+    coupon_code = serializers.CharField(max_length=200)
+    course_id = serializers.CharField(max_length=200)
+    date_placed = serializers.CharField(max_length=200)
+    edx_lms_user_id = serializers.IntegerField()
+    mode = serializers.CharField(max_length=200)
+    partner_sku = serializers.CharField(max_length=200)
+    titan_order_uuid = serializers.IntegerField()
+    edx_lms_username = serializers.CharField(max_length=200)
+
+    def create(self, validated_data):
+        # TODO: perform checks here
```

Comment:
Do more checks need to be performed here?

Reply:
Modifying to `save()` as per below comment